spec/features/friend_invitation_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 feature "friend invitation" do
-  scenario "user can invite a friend to join myflix" do
+  scenario "user can invite a friend to join myflix", js: true do
     jane = Fabricate :user
     sign_in_user jane
```

Comment:
I was a bit uncomfortable finishing this lesson and merging the PR with a failing spec. But will do if you say so :wink: